bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/utils/AudioStreamUtils.java

```diff
@@ -0,0 +1,75 @@
+/**
```

Comment:
@htreu thanks for your review. Have a look at this [discussion](https://github.com/eclipse/smarthome/pull/3598#discussion_r120611406). I suggested to use that util class, but we decided to be less dependant. That is the reason for my own implementation.